install_config/oab_broker_configuration.adoc

```diff
@@ -90,6 +90,14 @@ secrets: <6>
 <5> See xref:oab-config-broker[Broker Configuration] for details.
 <6> See xref:oab-config-secrets[Secrets Configuration] for details.
 
+[[install-config-oab-authenticating]]
+== Authenticating on Red Hat Connect Partner Registry
+
+Before configuring the Automation Broker, you must run the following command on all nodes of an {product-title} cluster to use the Red Hat Connect Partner Registry URL:
+----
+$ docker --config=/var/lib/origin/.docker login -u <registry-user> -p <registry-password> registry.connect.redhat.com
+----
+
```

Comment:
A few feedback points that I missed in my initial review:

There needs to be a space before the start of the code block and replaceable variables use underscores, not dashes. https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Reply:
From our conversation via Slack. I kept the dashes for consistency sake in the same file, but added the space above.